veocean-data-farming/df-background.md

```diff
@@ -0,0 +1,55 @@
+---
+description: Intuition and background on the Data Farming program.
+---
+# Data Farming Background
+
+It rewards OCEAN to liquidity providers (stakers) as a function of consume volume and liquidity. It’s like DeFi liquidity mining, but tuned for data consumption. DF’s aim is to achieve a minimum supply of data for network effects to kick in, and once the network flywheel is spinning, to increase growth rate.
+
+
+## Active Work to Drive APY
+
+Data Farming is not a wholly passive activity. The name of the game is to drive Data Consume Volume (DCV). High APYs happen only when there is sufficiently high DCV. High DCV means publishing and consuming truly useful datasets (or algorithms).
+
+Thus, if you really want to max out your APY:
+- create & publish datasets (and make $ in selling them) — or work with people who can
+- consume the datasets (to make $) — or work with people who can
+- go stake on them, and finally claim the rewards.
```

Comment:
Logically it would make sense if this was the second item on the list

Reply:
```
if this was the second item on the list
```
If "this" what?

Based on your feedback, I can't make out what you're recommending to me. What's the status quo, what should it be changed into?

Can you number the items and tell me the order you want them in?

Reply:
@jamiehewitt15 can we resolve this?

Reply:
I was referring to "go stake on them", my suggestion is that it should come before "consume the datasets (to make $) — or work with people who can". So the list looks something like this: 

1. create & publish datasets (and make $ in selling them) — or work with people who can
2. go stake on them
3. consume the datasets (to make $) — or work with people who can
4. finally claim the rewards.

Reply:
I failed to capture this although this is how other tutorials are setup.
I'll update right away.

Reply:
Resolved